python_modules/dagster/dagster/_core/definitions/run_request.py

```diff
@@ -111,6 +114,40 @@ def __new__(
         )
 
 
+# @whitelist_for_serdes
+# class NotABackfillRequest(  # TODO - decide on a new name
+#     NamedTuple(  # TODO - should this use the new dagster model?
+#         "_NotABackfillRequest",
+#         [
+#             ("asset_graph_subset", AssetGraphSubset),
+#             ("tags", PublicAttr[Optional[Mapping[str, str]]]),
+#             ("title", PublicAttr[Optional[str]]),
+#             ("description", PublicAttr[Optional[str]]),
+#         ],
+#     )
+# ):
+#     @property
+#     def asset_keys(self) -> AbstractSet[AssetKey]:
+#         return self.asset_graph_subset.asset_keys
+
+#     # TODO - maybe a from_asset_key_partition_key method?
+
+
+@whitelist_for_serdes
```

Comment:
had to switch to named tuple in stacked pr because of deserialization issue. need to investigate when have time